pdns/iputils.hh

```diff
@@ -261,6 +261,16 @@ union ComboAddress {
     return ret;
   }
 
+  bool isValid() const
```

Comment:
We added this isValid methods so we could easily test if the scope was currently set to something or not.